data/human/intro missions.txt

```diff
@@ -413,24 +413,112 @@ mission "Intro [2 Interceptor]"
+				accept
+	
+	on complete
+		conversation
+			`You land on <planet> and sell your cargo of minerals. You meet up with James and he asks you, "What did you think of mining?"`
+			choice
+				`	"I can't believe anyone would do that much work."`
+				`	"I can't believe how profitable that was."`
+			`	James nods and says, "When you're starting off, mining can be a fast way to build cash for your next ship. It's also a great way to explore the galaxy. Jobs always follow the trade routes and settled areas. Mining makes it profitable to explore the nooks and crannies of the hyperlanes. If the place you end up landing has an economy, they'll buy the minerals in your hold. Mainly though, it's about how you like to spend your time. Some captains like managing a trading fleet while others like to strike out on their own and fly their ship directly."`
+			`	"Mining can be complex compared to running jobs or trading commodities. Are there any questions you'd like to ask?"`
+
+			label choices
+			choice
+				`	"What equipment do I need to mine?"`
+					goto equipment
+				`	"Can my fleet help me mine?"`
+					goto fleet
+				`	"Where are safe places to mine?"`
+					goto safety
+				`	"How do you deal with pirates?"`
+					goto pirates
+				`	"How can I check the value of minerals?"`
+					goto value
+				`	"I don't have any more questions."`
+					goto end
+
+			label equipment
+			`	"Well, you need a weapon that does hull damage," James says." A beam laser will get the job done, but it will take awhile. The most valuable tool is an asteroid scanner. They're expensive, but they connect into your navigation and targeting systems to point you towards minable asteroids. You can also set them to find the most valuable asteroids which increases your profitability per run. After that, a tactical scanner will give you range to target and show how much damage you're doing. An afterburner is good for chasing down fast moving asteroids or getting away from pirates without leaving the system. A ramscoop will let you refuel while mining. You can extend your range and be ready for a quick escape."`
+			choice
+				`	"I can't fit all that on my ship!"`
+					goto shipmods
+				`	"That all makes sense."`
+					goto more
+			
+			label shipmods
+			`	James gives you a knowing smile as he answers, "Not in a factory build, you won't. But you'd be surprised how much space you can free up even in a Sparrow if you're willing to flexible about health and safety. Do you really need that much power? If you've got an outfitter on the planet where you're mining, do you need a hyperdrive, or even a shield regenerator? Maybe a cargo expansion would be more use." Seeing the look on your face he holds up his hands. "I'm not saying you should, of course, but you have options."`
```

Comment:
> Seeing the look on your face he holds up his hands. "I'm not saying you should, of course, but you have options."

You probably shouldn't say you have a look on your face; that assumes stuff about the player. And for me, that's a wrong assumption. I totally do that!

Reply:
Removed  the player's reaction:  He pauses briefly, perhaps considering what liability he's opened himself up for, then adds, "I'm not saying you should, of course, but you have options.